geonode_mapstore_client/client/js/components/DetailsPanel/DetailsLocations.jsx

```diff
@@ -0,0 +1,179 @@
+        }
+    }, [copied]);
+    return (
+        <div className={`gn-location-info ${expand ? 'fullWidth' : ''}`}>
+            <div className="bounds">
+                <div className="title">
+                    <Message msgId={"gnviewer.boundingBox"}/>
+                    <CopyToClipboard
+                        text={!isEmpty(extent) && wk.stringify(bboxToFeatureGeometry(extent))}
+                    >
+                        <Button
+                            variant="default"
+                            onClick={()=> setCopied(true)}>
+                            <FaIcon name="copy" />
+                        </Button>
+                    </CopyToClipboard>
+                </div>
+                <div className="bounds-row">
+                    <Message msgId="gnviewer.minLat"/>
+                    {miny?.toFixed(6)}
+                </div>
+                <div className="bounds-row">
+                    <Message msgId="gnviewer.minLon"/>
+                    {minx?.toFixed(6)}
+                </div>
+                <div className="bounds-row">
+                    <Message msgId="gnviewer.maxLat"/>
+                    {maxy?.toFixed(6)}
+                </div>
+                <div className="bounds-row">
+                    <Message msgId="gnviewer.maxLon"/>
+                    {maxx?.toFixed(6)}
+                </div>
+            </div>
+            <div className="center">
+                <div className="title">
+                    <Message msgId={"gnviewer.center"}/>
+                    <CopyToClipboard
+                        text={!isEmpty(center) && wk.stringify(center)}
+                    >
+                        <Button
+                            variant="default"
+                            onClick={()=> setCopied(true)}>
+                            <FaIcon name="copy" />
+                        </Button>
+                    </CopyToClipboard>
+                </div>
+                <div className="center-row">
+                    <Message msgId="gnviewer.centerLat"/>
+                    {get(center, 'geometry.coordinates.[1]')?.toFixed(6)}
+                </div>
+                <div className="center-row">
+                    <Message msgId="gnviewer.centerLon"/>
+                    {get(center, 'geometry.coordinates.[0]')?.toFixed(6)}
+                </div>
+            </div>
+        </div>
+    );
+};
+
+const DetailsLocations = ({ onSetExtent, fields, allowEdit} = {}) => {
+    const extent = get(fields, 'extent.coords');
+    const initialExtent = get(fields, 'initialExtent.coords');
+
+    const polygon = !isEmpty(extent) ? getPolygonFromExtent(extent) : null;
+    const center = !isEmpty(extent) && polygon ? turfCenter(polygon) : null;
+    const isDrawn = !isEqual(initialExtent, extent);
+
+    return (
+        <div className="gn-viewer-extent-map">
+            <BoundingBoxAndCenter center={center} extent={extent} expand={!allowEdit}/>
+            {allowEdit && <Map
```

Comment:
The map should be always visible, just the editing should be disabled based on permission and resource type

Reply:
Yes. Not needed when map should be visible for all resource